Add ip_fw_addtable.c to the ipfw kernel module build.

The file is appended alongside the other ip_fw_*.c sources in
sys/modules/ipfw/Makefile (or sys/netpfil/ipfw/Makefile depending on
the tree layout used for the ipfw.ko module).

--- a/sys/modules/ipfw/Makefile
+++ b/sys/modules/ipfw/Makefile
@@ -10,6 +10,7 @@ SRCS=	ip_fw2.c \
 	ip_fw_dynamic.c \
 	ip_fw_eaction.c \
 	ip_fw_log.c \
+	ip_fw_addtable.c \
 	ip_fw_main.c \
 	ip_fw_nat.c \
 	ip_fw_pfil.c \
